app/assets/stylesheets/mtl/extend/_chips.scss

```diff
@@ -1,3 +1,7 @@
 .chip {
   white-space: nowrap;
+  height: $chip-height;
+  font-size: $chip-font-size;
+  font-weight: $chip-font-weight;
+  line-height: $chip-line-height;
```

Comment:
Possible to add those to https://github.com/at-point/mtl/blob/master/lib/generators/mtl/templates/_variables.scss with the default values?